android/cli/commands/_buildModule.js

```diff
@@ -889,7 +891,15 @@ AndroidModuleBuilder.prototype.runModule = async function (cli) {
 
 	// Run the temp app.
 	this.logger.debug(__('Running example project...', tmpDir.cyan));
-	const buildArgs = [ process.argv[1], 'build', '-p', 'android', '-d', tmpProjectDir ];
+	let buildArgs = [ process.argv[1], 'build', '-p', 'android', '-d', tmpProjectDir ];
```

Comment:
We've discussed exposing these previously and one issue raised was that there would be no verification of chosen target/device ID until the app build has started, do you think that's an issue or maybe just a potential future improvement?

This change should also be done across ios/android at the same time preferably

Reply:
IMO, this is not a big issue as the target and device are not set by user but choosen from a list of avalaible ones detected.

The risk is only that if one device/emultator  shutdown/disconnect during build time before application build.

I think it should not be difficult to make a pre-verification but is that make sens ? the risk will stay the same, i mean if they get shutdown/disconnect after verification during module compilation, that's why I dont think it is much an issue.

Reply:
> This change should also be done across ios/android at the same time preferably

Sorry, I did completely forget this part, I think the Titanium SDK part is made for both android/ios (as it was requiring very small code change) but I forgot in this VSCode PR, having no mac or ios device may make this a bit hard for me to test.

Reply:
https://github.com/tidev/titanium_mobile/blob/6526d2d55c1989fb113572efb03950de87bfef01/iphone/cli/commands/_buildModule.js#L1072-L1088

Does not look very different logic, I will update the PR, but it wont be tested...